From 746e0e38b8853f55c4b4310afb517faa1899b623 Mon Sep 17 00:00:00 2001
From: barracuda156 <vital.had@gmail.com>
Date: Wed, 17 May 2023 14:37:00 +0800
Subject: [PATCH 1/6] Revert "Update methods to
 `dcheck{ToSend,Received}OrEmpty()`"

This reverts commit ace41c041175402779b1c05981b5dd7a48a4ee46.
---
 folly/io/async/fdsock/SocketFds.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git folly/io/async/fdsock/SocketFds.h folly/io/async/fdsock/SocketFds.h
index 965bdb5ac..620a5dd0a 100644
--- folly/io/async/fdsock/SocketFds.h
+++ folly/io/async/fdsock/SocketFds.h
@@ -83,12 +83,12 @@ class SocketFds final {
     DCHECK(!ptr_);
     return *this;
   }
-  SocketFds& dcheckReceivedOrEmpty() {
-    DCHECK(!ptr_ || std::holds_alternative<Received>(*ptr_));
+  SocketFds& dcheckReceived() {
+    DCHECK(std::holds_alternative<Received>(*ptr_));
     return *this;
   }
-  SocketFds& dcheckToSendOrEmpty() {
-    DCHECK(!ptr_ || std::holds_alternative<ToSend>(*ptr_));
+  SocketFds& dcheckToSend() {
+    DCHECK(std::holds_alternative<ToSend>(*ptr_));
     return *this;
   }
 
-- 
2.40.1

